### PR TITLE
Adds cardboard cutouts

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
rscadd: Cardboard cutouts have been added. Now you can set up advertisements for your clown mart.
/ :cl:

Added cardboard cutouts, created from five cardboard sheets. They can be picked up and moved around. Using a crayon or spray can on one allows you change its appearance to a variety of forms, such as clown, mime, and deathsquad officer. These cutouts are discolored and have a slightly brown tint to them.

They can be pushed over by interacting with them while on harm intent. Once pushed down, the cutout's appearance is reset and it has to be righted by activating it while holding it.

The Syndicate can buy a box of three adaptive cardboard cutouts for one telecrystal. These cutouts don't suffer from the discoloration and appear identical to what they're supposed to represent. Also included in the bundle is a crayon for changing their appearance.

_Adaptive cutouts are on the bottom, normal ones on the top_

**Token: #18722**

### CODE REVIEW

File: code/game/objects/items/cardboard_cutouts.dm [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+			desc = "A cardboat cutout of an assistant."
+			icon_state = "cutout_greytide"
+		if("Clown")
+			name = pick(clown_names)
+			desc = "A cardboard cutout of a clown. You get the feeling that it should be in a corner."
+			icon_state = "cutout_clown"
+		if("Mime")
+			name = pick(mime_names)
+			desc = "...(A cardboard cutout of a mime.)"
+			icon_state = "cutout_mime"
```
Comment (MrPerson) [inline code/game/objects/items/cardboard_cutouts.dm]: you're a tardboard

**Comment by GunHog** [comment]
Pictures, please.

File: code/game/objects/items/cardboard_cutouts.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/objects/items/cardboard_cutouts.dm]: no u

File: code/game/objects/items/cardboard_cutouts.dm [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	if(!new_appearance || !crayon || !user.canUseTopic(src))
+		return
+	user.visible_message("<span class='notice'>[user] gives [src] a new look.</span>", "<span class='notice'>Voila! You give [src] a new look.</span>")
+	crayon.use_charges(1)
+	crayon.check_empty(user)
+	if(!deceptive)
+		color = "#FFD7A7"
+	switch(new_appearance)
+		if("Assistant")
+			name = "[pick(first_names_male)] [pick(last_names)]"
```
Comment (RemieRichards) [inline code/game/objects/items/cardboard_cutouts.dm]: what are you insinuating boyo.

**Comment by CosmicScientist** [comment]
PR of the century.

File: code/game/objects/items/cardboard_cutouts.dm (hunk 2 above)
Comment (Xhuis) [inline code/game/objects/items/cardboard_cutouts.dm]: ?

**Comment by Xhuis** [comment]
OP updated with pictures.

File: code/game/objects/items/cardboard_cutouts.dm (hunk 2 above)
Comment (CosmicScientist) [inline code/game/objects/items/cardboard_cutouts.dm]: Yeah. Is ugly, discoloured, flimsy cardboard only good enough for one and not the other?

**Comment by Cheridan** [comment]
this feature is unfair to the colorblind :see_no_evil:

**Comment by demonfiren** [comment]
I wonder if three TCs isn't a bit steep for something you can identify by slapping it once.

Maybe if the syndie cutouts were able to be hooked up to remotes or outfitted with a grenade or shit like that.

**Comment by Xhuis** [comment]
OP was wrong. The bundle is just 1 TC.

**Comment by bgobandit** [comment]
should shooting targets be rolled into these

File: code/game/objects/items/cardboard_cutouts.dm [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+		if("Shadowling")
+			name = "Unknown"
+			desc = "A cardboard cutout of a shadowling."
+			icon_state = "cutout_shadowling"
+		if("Xenomorph")
+			name = "alien hunter ([rand(1, 999)])"
+			desc = "A cardboard cutout of a xenomorph."
+			icon_state = "cutout_fukken_xeno"
+			if(prob(25))
+				alpha = 75 //Spooky sneaking!
```
Comment (AnturK) [inline code/game/objects/items/cardboard_cutouts.dm]: That's some advanced cardboard technology

File: code/game/objects/items/cardboard_cutouts.dm [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+	w_class = 4
+	var/list/possible_appearances = list("Assistant", "Clown", "Mime", "Traitor", "Nuke Op", "Cultist", "Clockwork Cultist", "Revolutionary", "Wizard", "Shadowling", "Xenomorph", "Swarmer", \
+	"Ash Walker", "Deathsquad Officer", "Ian") //Possible restyles for the cutout; add an entry in change_appearance() if you add to here
+	var/pushed_over = FALSE //If the cutout is pushed over and has to be righted
+	var/deceptive = FALSE //If the cutout actually appears as what it portray and not a discolored version
+
+/obj/item/cardboard_cutout/attack_hand(mob/living/user)
+	if(user.a_intent == "help" || pushed_over)
+		return ..()
+	user.visible_message("<span class='warning'>[user] pushes over [src]!</span>", "<span class='danger'>You push over [src]!</span>")
```
Comment (AnturK) [inline code/game/objects/items/cardboard_cutouts.dm]: Probably should also react to bullets in the same way.

**Comment by InsaneHyena** [comment]
It's a nice code and all, but why would traitors want to buy cardboard cutouts? What purpose may it serve?

**Comment by Xhuis** [comment]
Hilarity. Same reason as the Syndicate balloon. It might also serve as a metabreaker to the trigger-happy. See a shadowling cutout in maintenance? Some people would just run and screech "SLINGS" over radio.

**Comment by optimumtact** [comment]
Do these block bullets and if you shoot them do they get knocked over? Can they be deconstructed? How do I get rid of them?

**Comment by Xhuis** [comment]
1. In the next commit they will block bullets and will be knocked over if a `prob(bullet damage)` check passes.
2. They cannot be deconstructed but should be burnable.
3. See above.

**Pull request merged**